### PR TITLE
Pbi 466 donor cards

### PR DESCRIPTION
**Pull request by adphillips2017**

### CODE REVIEW

File: projects/ccf-eui/src/app/app.component.html [hunk 1]
```diff
@@ -37,6 +37,7 @@
       <ccf-results-browser
         [dataLoading]="spinnerActive$ | async"
         [data]="data.listData$ | async"
```
Comment (bherr2) [inline projects/ccf-eui/src/app/app.component.html:39]: @adphillips2017 Is this still needed?

Comment (adphillips2017) [reply projects/ccf-eui/src/app/app.component.html:39]: No, but I thought you said to make it so that we could easily switch back to the old version?

Comment (bherr2) [reply projects/ccf-eui/src/app/app.component.html:39]: In your follow-up PR, you can remove the pieces that are no longer needed. I just wanted that stuff to stay until we had your stuff dropped in.

**Comment by bherr2** [comment]
Just few code smells to go!

**Pull request merged**